adam-core/src/main/scala/org/bdgenomics/adam/rdd/read/AlignmentRecordRDDFunctions.scala

```diff
@@ -44,8 +44,20 @@ import org.bdgenomics.utils.misc.Logging
 import org.seqdoop.hadoop_bam.SAMRecordWritable
 import scala.annotation.tailrec
 import scala.language.implicitConversions
+import scala.math.{ abs, min }
 import scala.reflect.ClassTag
 
+private object SequenceIndexOrdering extends Ordering[(Int, Long)] {
```

Comment:
You are right! When I wrote the code, I didn't realize that I had to explicitly import the `scala.math.Ordering` implicit orderings. When I got the compile error, I then just decided to grunt the ordering out. Anywho, patching that now!